src/components/panel/panel.component.tsx

```diff
@@ -43,7 +48,14 @@ const getTitle = (
   return title as string;
 };
 
-export const Panel = ({ closePanel, objectType, uid }: PanelProps) => {
+export const Panel = ({
+  closePanel,
+  objectType,
+  uid,
+  draggedObject,
+  newObject,
```

Comment:
imo `newObject` isn't descriptive of what it is, `newContentObject`? `addedContentObject`?

same as `setObject` is that `updateSetContent` ?